src/arload.py

```diff
@@ -123,6 +125,8 @@ def optarg(argflag, argv, pypname):
         if o in ('-h', '--help'): msgs.usage(None)
         elif o in ('-c', '--cpus'): argflag['run']['ncpus'] = a
         elif o in ('-v', '--verbose'): argflag['out']['verbose'] = int(a)
+    for a in arg:
```

Comment:
Except it isn't working that way (and I'm not sure how to debug it), e.g.

pypit lris_b_ls_600_4000_d560.red --use_masters --help

(Pdb) opt
[]
(Pdb) arg
['lris_b_ls_600_4000_d560.red', '--use_masters', '--help']